src/com/yetanalytics/lrs_admin_ui/functions/http.cljs

```diff
@@ -15,18 +15,21 @@
                   (format "%s%s/statements"
                           (if (some? proxy-path) proxy-path "")
                           xapi-prefix))
-        param-map (cond-> {:unwrap_html true}
+        param-map (cond-> {:limit @(subscribe [:browser/get-batch-size])}
                     (some? params)
-                    (merge (uri/query-map params)))
+                    (merge
+                     (if (string? params)
+                       (uri/query-map params)
+                       params)))
         params' (uri/map->query-string param-map)]
     (format "%s%s?%s" server-host path' params')))
 
 (defn extract-params
   "return a map of parameters, unencoded and cleaned, from an xapi url,
-  excluding unwrap"
+  excluding limit"
   [address]
   (-> (uri/query-map (uri/uri address) {:keywordize? false})
-      (dissoc "unwrap_html")))
+      (dissoc "limit")))
 
```

Comment:
What happened to `unwrap_html`?

Reply:
its no longer relevant. we are not using html mode. this is raw json API now.